repos/exporter/lib/exporter-stack.ts

```diff
@@ -64,7 +64,7 @@ export class ExporterStack extends cdk.Stack {
             logRetention: 180,
         });
 
-        this.addAlarmsToLambda(stage, exporterFunction, `exporter-${stage}`, 30000);
+        this.addAlarmsToLambda(stage, exporterFunction, `exporter-${stage}`, 300000);
```

Comment:
Can you explain this change, please? What does the `300000` mean or do?

Reply:
This takes the duration alarm from 30 seconds, to 5 minutes